### PR TITLE
SISRP-20669 - Add 'Terms' column to enrollment lists

### PR DESCRIPTION
**Pull request by redconfetti**
https://jira.berkeley.edu/browse/SISRP-20669

### CODE REVIEW

File: app/models/rosters/common.rb [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# encoding: UTF-8
```
Comment (paulkerschen) [inline app/models/rosters/common.rb]: @redconfetti Why is the encoding comment needed?

File: public/dummy/json/campus_rosters.json [hunk 2]
```diff
@@ -92,8 +94,9 @@
       "last_name": "LBBERRR",
       "login_id": "333333",
       "majors": [
-        "Landscape Architecture BA"
+        "Education PhD"
       ],
+      "terms_in_attendance": 'G',
```
Comment (paulkerschen) [inline public/dummy/json/campus_rosters.json]: @redconfetti ANGRY RED single quotes in JSON

File: spec/models/rosters/common_spec.rb [hunk 3]
```diff
@@ -1,3 +1,4 @@
+# encoding: UTF-8
```
Comment (paulkerschen) [inline spec/models/rosters/common_spec.rb]: @redconfetti Same encoding question here. If it's the em dash — that requires it, might be preferable to refer to it by codepoint: `"\u2014"`.

Comment (redconfetti) [inline spec/models/rosters/common_spec.rb]: @paulkerschen It is the em dash. Thanks for the tip. I didn't know that one of those codes would be interpreted via JSON properly.

File: src/assets/javascripts/angular/controllers/pages/rosterController.js [hunk 4]
```diff
@@ -19,6 +20,10 @@ angular.module('calcentral.controllers').controller('RosterController', function
     return (student.section_ccns.indexOf($scope.searchSection.ccn) !== -1);
   };
 
+  $scope.allStudentsCount = function() {
+    return _.get($scope, 'students.length', 0);
+  };
+
```
Comment (paulkerschen) [inline src/assets/javascripts/angular/controllers/pages/rosterController.js]: @redconfetti Did this function sneak in from a different branch? I don't see it in use anywhere.

File: spec/models/rosters/campus_spec.rb [hunk 5]
```diff
@@ -71,6 +51,7 @@
       expect(feed[:students].length).to eq 2
 
       student = feed[:students][0]
+      puts "student: #{student.inspect}"
```
Comment (paulkerschen) [inline spec/models/rosters/campus_spec.rb]: Stray puts, @redconfetti

**Pull request merged**